app/assets/stylesheets/noscript.scss

```diff
@@ -7,3 +7,7 @@
 .check-all-none {
   display: none;
 }
+
+.cookies-consent-banner {
+  display: none;
+}
```

Comment:
Shall we also hide the "Manage cookies" link/button which appears in the footer?